lumicks/pylake/file.py

```diff
@@ -48,6 +49,12 @@ def _check_file_format(self):
         if ff_version not in File.SUPPORTED_FILE_FORMAT_VERSIONS:
             raise Exception(f"Unsupported Bluelake file format version {ff_version}")
 
+        self.blacklist = {"Calibration": "file.force1x.calibration",
+                          "Marker": "file.markers",
+                          "FD Curve": "file.fdcurves",
+                          "Kymograph": "file.kymos",
+                          "Scan": "file.scans"}
```

Comment:
`blacklist` doesn't really fit as a name for the use in the printing function below.

Reply:
Agree. Renamed it to ```redirect_list``` since we're basically redirecting to an API class.